CONTRIBUTING.md

```diff
@@ -191,12 +191,10 @@ The documentation for this website is stored alongside the source, in the [docs
 
 The microsite generation requires a specific scala version that might
 differ from the project's one. We strongly suggest to check the CI's
-workflow to discover the proper version:
-[CI scala-version matrix](https://github.com/typelevel/cats/blob/main/.github/workflows/ci.yml#L230) and [CI makeMicrosite
-command](https://github.com/typelevel/cats/blob/main/.github/workflows/ci.yml#L281)
+workflow to discover the proper version at [CI site scala-version matrix](https://github.com/typelevel/cats/blob/v2.9.0/.github/workflows/ci.yml#L418-L425).
 
 At the moment the command is:
-`sbt ++2.12.15 docs/tlSite`
+`sbt ++2.13.10 docs/tlSite`
```

Comment:
Actually this should be fine now. The Scala version for build and docs are coupled.
```suggestion
`sbt docs/tlSite`
```

Reply:
Made some further changes to reflect that and make the section more concise